pkg/platform/aws/template.go

```diff
@@ -139,6 +139,14 @@ module "worker-pool-{{ $index }}" {
   os_version            = "{{ $pool.OSVersion }}"
   {{- end }}
 
+  {{- if $pool.Labels }}
+  labels                = "{{ $pool.Labels }}"
```

Comment:
It's a value separated by new line, so the alignment should "reset".
```suggestion
  labels = "{{ $pool.Labels }}"
```